src/poetry/console/commands/config.py

```diff
@@ -68,15 +68,16 @@ def unique_config_values(self) -> dict[str, tuple[Any, Any]]:
             ),
             "virtualenvs.path": (str, lambda val: str(Path(val))),
             "virtualenvs.prefer-active-python": (boolean_validator, boolean_normalizer),
+            "virtualenvs.prompt": (str, lambda val: str(val)),
```

Comment:
I realize you just moved the line but why is this `lambda val: str(val)` instead of `str` like the first element?

Reply:
Probably, a copy & paste (& edit) of another line. Since I had already touched the line anyway and there's no risk in the change, I adjusted it straight away.